doc/source/glossary.rst

```diff
@@ -214,126 +225,58 @@ Glossary
        arrays are :term:`view`\ s.
 
 
-   big-endian
-       When storing a multi-byte value in memory as a sequence of bytes, the
-       sequence addresses/sends/stores the most significant byte first (lowest
-       address) and the least significant byte last (highest address). Common in
-       micro-processors and used for transmission of data over network protocols.
+   `big-endian <https://en.wikipedia.org/wiki/Endianness>`_
```

Comment:
Are you sure these work as targets for `` :term:`big-endian` ``?

Reply:
You can try it: Such a link appears in the > entry; it takes you to the big-endian glossary entry, which is itself a working link to the Wikipedia article; similarly for little.

Reply:
You're right, this seems to work. As a reader of the HTML docs though, it's not at all obvious to me that the definitions are links, especially if they're already highlighted due to being the focused link target.

For that reason, I think a simple description of `see ...` might be clearer, especially since we use that elsewhere in the glossary.

Reply:
Added in 563f5c27a.